DOC-2009: Added TINY-9688 to the release notes. (#2782)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

The following premium plugin has been announced as reaching its end-of-life:

=== Real-Time Collaboration

> {productname}'s xref:rtc-introduction.adoc[Real-time Collaboration (RTC)] plugin was officially deactivated on December 31, 2023, and is no longer available for purchase.

Reviewer: ```suggestion
{productname}'s xref:rtc-introduction.adoc[Real-time Collaboration (RTC)] plugin will be deactivated on December 31, 2023, and is no longer available for purchase.
```

?

Author: @TheSpyder thanks for picking up that error 😬
